src/components/Trade/TradeInput.tsx

```diff
@@ -491,7 +491,7 @@ export const TradeInput = ({ history }: RouterProps) => {
                   ? 'red'
                   : 'blue'
               }
-              isLoading={isSubmitting || isSendMaxLoading}
```

Comment:
I had initial concerns about this but it makes sense... kind of. `isSubmitting` is triggered on async form submit operations, and there's no way to programmatically set it back to false.

So while it does make sense not to react on this field for loading states, the implication of this is that there's now a user-noticeable delay between clicking "Preview" and being routed to the next route, which is not *not* instant.

Reply:
I havent seen a delay

Reply:
Added state to track this onSubmit button only